test/components/figure_test.rb

```diff
@@ -0,0 +1,32 @@
+require 'component_test_helper'
+
+class FigureTest < ComponentTestCase
+  def component_name
+    "figure"
+  end
+
+  test "fails to render a figure when no data is given" do
+    assert_raise do
+      render_component({})
+    end
+  end
+
+  test "fails to render a figure when a href is provided but alt text isn't" do
```

Comment:
Not sure about this. Ideally – yes, but we aren't in control of the content and this may silently hide images that content editors expect to see. 🤔